### ISSUE TITLE
[addon-storysource] Remove knobs from source code

### ISSUE THREAD

**Issue by ivoiv** [storybookjs/storybook#9801]
**Is your feature request related to a problem? Please describe.**
When using addons-storysource, it also shows knobs as part of the story code. This makes the story code difficult to read and copy-paste to actual code.

Knobs are an implementation detail of the story and not relevant to the actual use of the component.

**Describe the solution you'd like**
action-knobs functions should be interpolated/removed

```
isLoading={boolean("isLoading", true)}
loadingText={text("loadingText", "Only visible when isLoading")}

```
should become
````
isLoading
loadingText="Only visible when isLoading"
````

**Describe alternatives you've considered**
Writing a large comment explaining what to remove to make the code usable in production...

**Are you able to assist bring the feature to reality?**
Can try

**Additional context**

**Comment by MarcusNotheis** [comment]
Leaving a comment to keep that issue open - would love to see this as part of storybook.

**Comment by juanpprieto** [comment]
Would love this too! - would be great to also remove knobs code from the generic `code` view on the Docs

**Comment by gca-bliss** [comment]
Was also looking into this. Knobs code is pretty useless for the enduser, and to have a way of just copy the output would bring a lot of benefits for non-coder users such as PO's or QA's

**Comment by shilman** [comment]
Hi gang, We’ve just released addon-controls in 6.0-beta!

Controls are portable, auto-generated knobs that are intended to replace addon-knobs long term.

- [Controls README](https://github.com/storybookjs/storybook/tree/next/addons/controls)
- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)

Please upgrade and try them out today. Thanks for your help and support getting this stable for release!

**Comment by Lav39** [comment]
> Hi gang, We’ve just released addon-controls in 6.0-beta!
> 
> Controls are portable, auto-generated knobs that are intended to replace addon-knobs long term.
> 
> * [Controls README](https://github.com/storybookjs/storybook/tree/next/addons/controls)
> * [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)
> 
> Please upgrade and try them out today. Thanks for your help and support getting this stable for release!

Does not work at all. It simply crashes with error `(0 , _api.useArgs) is not a function`.

**Comment by shilman** [comment]
@Lav39  did you upgrade all your storybook packages?

**Comment by Lav39** [comment]
> @Lav39 did you upgrade all your storybook packages?

yes, when i dive deep, i see `useArgs` being exported from `@storybook/api` but somehow it is not able available at runtime.

**Comment by shilman** [comment]
Try clearing your node modules and possibly regenerating your lock file. Or use “yarn why”. It’s def using an old version of the package for some reason

**Comment by Lav39** [comment]
> Try clearing your node modules and possibly regenerating your lock file. Or use “yarn why”. It’s def using an old version of the package for some reason
 I tried updating packages but not luck. here is how my package.json look like
`
........
    "@storybook/addon-actions": "^5.3.19",
    "@storybook/addon-controls": "^6.0.0-beta.15",
    "@storybook/addon-docs": "^5.3.19",
    "@storybook/addon-knobs": "^5.3.18",
    "@storybook/addon-links": "^5.3.19",
    "@storybook/addon-storysource": "^5.3.19",
    "@storybook/addons": "^5.3.19",
    "@storybook/react": "^5.3.19",
........
`

**Comment by Lav39** [comment]
it fails with fresh new installation also.

i followed following two links to setup storybook and controls add-on:
https://storybook.js.org/docs/guides/guide-react/
https://github.com/storybookjs/storybook/tree/next/addons/controls

**Comment by shilman** [comment]
@Lav39  please share a repro

**Comment by Lav39** [comment]
> @Lav39 please share a repro

https://github.com/Lav39/strorybook-control

**Comment by shilman** [comment]
For anybody who is interested in Controls but don't know where to start, I've created a quick & dirty step-by-step walkthrough to go from a fresh CRA project to a working demo. Check it out:

=> [Storybook Controls w/ CRA & TypeScript](https://gist.github.com/shilman/69c1dd41a466bae137cc88bd2c6ef487)

There are also some "knobs to controls" migration docs in the Controls README:

=> [How do I migrate from addon-knobs?](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)

**Comment by jdmacmurtrie** [comment]
@Lav39 I've been having the same issue.  I ran the storybook in the repo you provided, and it worked.  I see all of your storybook versions are `"^6.0.0-beta.20"`.  Is this how you got it to work?

**Comment by shilman** [comment]
@Lav39 I cloned your repo, ran the following, and it worked fine on my machine:

```
npx sb@next upgrade --prerelease
```

**Comment by zacnomore** [comment]
It seems that this issue has been solved by controls, @ivoiv can we close it?

**Comment by shilman** [comment]
Solved via controls https://storybook.js.org/docs/react/essentials/controls
